### PR TITLE
End to end error message

### PR DESCRIPTION
**Pull request by haaakon**

### CODE REVIEW

File: Digipost/POSAttachment.h [hunk 1]
```diff
@@ -47,6 +47,8 @@ extern NSString *const kAuthenticationLevelPassword;
 @property (nonatomic, retain) POSDocument *document;
 @property (nonatomic, retain) POSInvoice *invoice;
 @property (nonatomic, retain) NSString *origin;
+@property (nonatomic, retain) NSNumber *endToEndEncrypted;
+@property (nonatomic, retain) NSNumber * userKeyEncrypted;
```
Comment (lillesand) [inline Digipost/POSAttachment.h]: I take it the space between the `*` and the variable name is optional?

File: Digipost/nb.lproj/Localizable.strings [hunk 2]
```diff
@@ -507,11 +507,17 @@
...
 /* No comment provided by engineer. */
-"" = "";
-
-/* No comment provided by engineer. */
 "not connected to internet alert message" = "Koble til, og prøv igjen";
 
 /* No comment provided by engineer. */
 "not connected to internet alert title" = "Du er ikke koblet til internett";
 
+/* No comment provided by engineer. */
```
Comment (lillesand) [inline Digipost/nb.lproj/Localizable.strings:515]: I suggest we remove all of these comments. "No comment provided by engineer" :grinning:

**Comment by lillesand** [comment]
Review my two comments and merge at will

File: Digipost/nb.lproj/Localizable.strings (hunk 2 above)
Comment (haaakon) [inline Digipost/nb.lproj/Localizable.strings:515]: :hankey: 
Autogenerated code from Linguan, no need to remove

File: Digipost/POSAttachment.h (hunk 1 above)
Comment (haaakon) [inline Digipost/POSAttachment.h]: Yes, and i've turned off clang formatting when doing these mergings, can do a commit with fix for it, to make it look nicer.

**Pull request merged**